.env.example

```
@@ -12,4 +12,4 @@ MONGODB_URI=mongodb://localhost:27017/monew
 AWS_ACCESS_KEY=your_access_key
 AWS_SECRET_KEY=your_secret_key
 AWS_REGION=ap-northeast-2
-S3_BUCKET_NAME=monew-backup
\ No newline at end of file
+AWS_S3_BUCKET=monew-backup
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)